src/rsz/src/Resizer.cc

```diff
@@ -1810,8 +1810,14 @@ void Resizer::reportDontUse() const
   if (dont_use_.empty()) {
     logger_->report("  none");
   } else {
+    std::set<dbMaster*> cells;
+
     for (auto* cell : dont_use_) {
-      logger_->report("  {}", cell->name());
+      cells.insert(db_network_->staToDb(cell));
```

Comment:
I think `if(isLinkCell(cell))` also filters out the per-corner copies

Reply:
@gadfort this seems correct

Reply:
done